src/main/resources/archetype-resources/run.sh

```diff
@@ -21,6 +21,5 @@ APPHOME=home
 . apphome.sh
 
 mvn exec:java -Dapp.home=$APPHOME \
-              -Dconfig.file=$APPHOME/cfg/application.conf \
```

Comment:
I used the conf file in one project (and removed it since) but generally it is not necessary; a properties file will suffice.